src/components/IWantUrNFTForm.vue

```diff
@@ -92,6 +92,8 @@ import { NFTGet } from '@/common/NFTget';
 import {notifyGMNHUser} from '@/composables/airtable';
 import {getQuestionUserWalletId, generateTicketDetailLink, formatTicketDetailLink} from '@/composables/pnftInteractions'
 import {emailTypeAnswered, emailTypeResponder} from '@/composables/emailjs'
+import { uploadImage} from '@/composables/gmnh-service';
```

Comment:
super minor: i know the semicolon is not an issue (but since other imports don't have it, could remove)

Reply:
cool ill fix in next PR